pkg/configuration/config_manager/handler_util.go

```diff
@@ -189,13 +189,25 @@ func GetSupportReloadConfigSpecs(configSpecs []appsv1alpha1.ComponentConfigSpec,
 	return reloadConfigSpecMeta, nil
 }
 
-func FilterSubPathVolumeMount(metas []ConfigSpecMeta, volumes []corev1.VolumeMount) []ConfigSpecMeta {
+func FilterSupportReloadActionConfigSpecs(metas []ConfigSpecMeta, volumes []corev1.VolumeMount) []ConfigSpecMeta {
 	var filtered []ConfigSpecMeta
 	for _, meta := range metas {
 		v := FindVolumeMount(volumes, meta.ConfigSpec.VolumeName)
-		if v == nil || v.SubPath == "" || meta.ReloadType == appsv1beta1.TPLScriptType {
+		if isSyncReloadAction(meta.ConfigSpecInfo) || !isSubPathMount(v) {
 			filtered = append(filtered, meta)
 		}
```

Comment:
This check will pass if isSubPathMount(v) is true, but isSyncReloadAction(meta.ConfigSpecInfo) return true?